<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Perlea Creator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="image">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>461</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionOuvrir"/>
    <addaction name="actionEnregistrer"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuGrilles">
    <property name="title">
     <string>Cr√©er une grile</string>
    </property>
    <addaction name="actionLoom"/>
    <addaction name="actionPeyote"/>
    <addaction name="actionSquare"/>
    <addaction name="actionBrick"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="actionManuel"/>
    <addaction name="actionApropos"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuGrilles"/>
   <addaction name="menuAide"/>
  </widget>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionOuvrir">
   <property name="text">
    <string>Ouvrir image...</string>
   </property>
  </action>
  <action name="actionLoom">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Loom...</string>
   </property>
  </action>
  <action name="actionPeyote">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Peyote...</string>
   </property>
  </action>
  <action name="actionEnregistrer">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Enregistrer...</string>
   </property>
  </action>
  <action name="actionSquare">
   <property name="text">
    <string>Square...</string>
   </property>
  </action>
  <action name="actionBrick">
   <property name="text">
    <string>Brick...</string>
   </property>
  </action>
  <action name="actionManuel">
   <property name="text">
    <string>Manuel</string>
   </property>
  </action>
  <action name="actionApropos">
   <property name="text">
    <string>A propos de</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
